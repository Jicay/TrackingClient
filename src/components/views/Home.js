import React, {Component} from "react";

import Image from "./Image";

export default class Home extends Component {
    constructor(props) {
        super(props);
    }

    componentDidMount() {
        const {onLoad} = this.props;
        console.log("Home " + this.props.height + " " + this.props.width);
        onLoad();
    }

    render() {
        return (
            <div className="home container" id="page" >
                <div className="row  justify-content-center">
                    <Image image={'916/43406628822_cffd9b699e_b'}/>
                    <Image image={'65535/48877432621_ff264905ac_b'}/>
                </div>
                <div className="row justify-content-center">
                    <Image image={'65535/48876898518_9f01d744c4_b'}/>
                    <Image image={'65535/48877431621_5b73d110a6_b'}/>
                </div>
            </div>
        );
    }
}