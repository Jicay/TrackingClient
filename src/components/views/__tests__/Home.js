import React from 'react';
import renderer from 'react-test-renderer';
import Home from "../Home";
import {MemoryRouter} from "react-router";

it('renders correctly', () => {
    const tree = renderer
        .create(<MemoryRouter><Home onLoad={jest.fn()}/></MemoryRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});