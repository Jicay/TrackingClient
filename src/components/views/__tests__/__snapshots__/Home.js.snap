// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="home container"
  id="page"
>
  <div
    className="row  justify-content-center"
  >
    <div
      className="col align-content-center"
    >
      <a
        href="/images/916-43406628822_cffd9b699e_b"
        onClick={[Function]}
      >
        <img
          className="img-fluid"
          src="https://live.staticflickr.com/916/43406628822_cffd9b699e_b.jpg"
        />
      </a>
    </div>
    <div
      className="col align-content-center"
    >
      <a
        href="/images/65535-48877432621_ff264905ac_b"
        onClick={[Function]}
      >
        <img
          className="img-fluid"
          src="https://live.staticflickr.com/65535/48877432621_ff264905ac_b.jpg"
        />
      </a>
    </div>
  </div>
  <div
    className="row justify-content-center"
  >
    <div
      className="col align-content-center"
    >
      <a
        href="/images/65535-48876898518_9f01d744c4_b"
        onClick={[Function]}
      >
        <img
          className="img-fluid"
          src="https://live.staticflickr.com/65535/48876898518_9f01d744c4_b.jpg"
        />
      </a>
    </div>
    <div
      className="col align-content-center"
    >
      <a
        href="/images/65535-48877431621_5b73d110a6_b"
        onClick={[Function]}
      >
        <img
          className="img-fluid"
          src="https://live.staticflickr.com/65535/48877431621_5b73d110a6_b.jpg"
        />
      </a>
    </div>
  </div>
</div>
`;
